querqy-for-lucene/querqy-lucene/src/main/java/querqy/lucene/contrib/rewrite/wordbreak/GermanDecompoundingMorphology.java

```diff
@@ -0,0 +1,194 @@
+package querqy.lucene.contrib.rewrite.wordbreak;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+
+import java.util.Collections;
+
+/**
+ * <p>Morphological compounding patterns for German.</p>
+ * <p>The patterns where taken from: S. Langer. 1998. Zur Morphologie und Semantik von Nominalkomposita. Tagungsband
```

Comment:
Typo

Reply:
Argh, I'd simply correct this after the merge. Thank you!